packages/serve/package.json

```diff
@@ -14,14 +14,14 @@
     "@webpack-cli/utils": "0.2.2",
     "chalk": "2.4.2",
     "cross-spawn": "6.0.5",
-    "inquirer": "6.5.1",
-    "webpack-dev-server": "^3.8.1"
+    "inquirer": "6.5.1"
   },
   "devDependencies": {
     "@types/cross-spawn": "6.0.0",
     "@types/inquirer": "6.5.0",
     "@types/node": "12.7.2",
-    "typescript": "3.5.3"
+    "typescript": "3.5.3",
+    "webpack-dev-server": "^3.8.2"
```

Comment:
I think it should be in `dependencies` because this package can't be work without `webpack-dev-server`

Reply:
@evilebottnawi I thought we wanted it as a peer dependency like I did here:

https://github.com/webpack/webpack-cli/blob/d7799ade420cc31cfe29d1ca1e8cd58368f6a08a/packages/serve/package.json#L26-L33

Then we also need it as a dev dependency just so that we can test it.

Reply:
@webpack-cli/serve require `webpack-dev-server` to use so we should put it in `dependencies`